src/libs/ReportActionsUtils.js

```diff
@@ -174,6 +175,35 @@ function getLastVisibleMessageText(reportID, actionsToMerge = {}) {
         .substring(0, CONST.REPORT.LAST_MESSAGE_TEXT_MAX_LENGTH);
 }
 
+/**
+ * Filter out any reportActions which should not be displayed
+ * Then, sort by their created timestamp first, and reportActionID second
+ * This is a combination of getSortedReportActions() and filterReportActionsForDisplay()
+ *
+ * @param {Array} reportActions
+ * @returns {Array}
+ */
+function getSortedReportActionsForDisplay(reportActions) {
+    // HACK ALERT: We're temporarily filtering out any reportActions keyed by sequenceNumber
+    // to prevent bugs during the migration from sequenceNumber -> reportActionID
+    const filteredReportActions = _.filter(reportActions, (reportAction, key) => {
+        if (!reportAction) {
+            return false;
+        }
+
+        if (String(reportAction.sequenceNumber) === key) {
+            Log.info('Front-end filtered out reportAction keyed by sequenceNumber!', false, reportAction);
+            return false;
+        }
+
+        return true;
+    });
+
+    let sortedReportActions = getSortedReportActions(filteredReportActions, true);
+    sortedReportActions = filterReportActionsForDisplay(sortedReportActions);
+    return sortedReportActions;
```

Comment:
@tienifr Can we just return `filterReportActionsForDisplay(sortedReportActions)` rather than assign it to variable again?

Reply:
@mollfpr I just updated. Thanks for pointing out it